.github/workflows/release.yml

```diff
@@ -6,6 +6,14 @@ on:
       - v*
 
 jobs:
+  notify-runner:
```

Comment:
Ideally this should depend on the `release` job passing before notifying.

Reply:
Done, thanks👍